packages/react-composites/tests/browser/call/hermetic/HorizontalGalleryResizing.test.ts

```diff
@@ -0,0 +1,165 @@
+    expect(await horiztonalTiles.evaluate((e) => (e as HTMLDivElement).clientWidth)).toBeGreaterThanOrEqual(120);
+    expect(await horiztonalTiles.evaluate((e) => (e as HTMLDivElement).clientWidth)).toBeLessThanOrEqual(215);
+  });
+  test('resize should appropriately add tiles if room', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
+
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    addVideoStream(paul, true);
+    const fiona = defaultMockRemoteParticipant('Fiona Harper');
+    addVideoStream(fiona, true);
+    const reina = defaultMockRemoteParticipant('Reina Takizawa');
+    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
+
+    const participants = [
+      paul,
+      defaultMockRemoteParticipant('Eryka Klein'),
+      fiona,
+      defaultMockRemoteParticipant('Pardeep Singh'),
+      reina,
+      vasily,
+      defaultMockRemoteParticipant('Luciana Rodriguez'),
+      defaultMockRemoteParticipant('Antonie van Leeuwenhoek'),
+      defaultMockRemoteParticipant('Gerald Ho')
+    ];
+
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+    // set the initial viewPort size to activate the verticalGallery.
+    await page.setViewportSize({ width: 600, height: 600 });
+    // wait for the tiles to load
+    await waitForSelector(page, dataUiId(IDS.horizontalGalleryVideoTile));
+
+    // check initial tile number to be correct.
+    expect(await page.locator(dataUiId(IDS.horizontalGalleryVideoTile)).count()).toBe(2);
+
+    // resize the window.
+    await page.setViewportSize({ width: 800, height: 600 });
+
+    // verify that we added a tile with the extra spacing.
+    expect(await page.locator(dataUiId(IDS.horizontalGalleryVideoTile)).count()).toBe(3);
+  });
+  test('resize should appropriately remove tiles if needed', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
+
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    addVideoStream(paul, true);
+    const fiona = defaultMockRemoteParticipant('Fiona Harper');
+    addVideoStream(fiona, true);
+    const reina = defaultMockRemoteParticipant('Reina Takizawa');
+    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
+
+    const participants = [
+      paul,
+      defaultMockRemoteParticipant('Eryka Klein'),
+      fiona,
+      defaultMockRemoteParticipant('Pardeep Singh'),
+      reina,
+      vasily,
+      defaultMockRemoteParticipant('Luciana Rodriguez'),
+      defaultMockRemoteParticipant('Antonie van Leeuwenhoek'),
+      defaultMockRemoteParticipant('Gerald Ho')
+    ];
+
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+    // set the initial viewPort size to activate the verticalGallery.
```

Comment:
Did you mean activate the horizontalGallery?

Reply:
in the other comments in this test file as well?

Reply:
ah the comments are artifacts... will update